### PR TITLE
Task/remove auth endpoints

### PR DESCRIPTION
**Pull request by orndorffgrant**
# What does this do
This removes all the old auth endpoints in the orange code and the associated tests. Since we are now supporting generic JWT based authentication from an external identity provider/jwt issuer, this can all be classified as dead code.

# JIRA tickets this addresses
https://jira.amida-tech.com/browse/ORANGE-467

# How to test it
Use the app and verify nothing is broken. Note: the extAuth methods in the app still refer to auth endpoints in orange-api, these will need to be changed to point to the auth microservice external auth endpoints.

### CODE REVIEW

File: lib/controllers/auth.js [hunk 1]
```diff
@@ -1,80 +0,0 @@
-"use strict";
-
```
Comment (puckybreg) [inline lib/controllers/auth.js]: @ElijahGeorge Are any parts of this file still important to hold onto or are we good with deleting them.

**Comment by orndorffgrant** [comment]
@ElijahGeorge

File: lib/controllers/auth.js (hunk 1 above)
Comment (orndorffgrant) [reply lib/controllers/auth.js]: 👍 I'll add the facebook auth back in

**Comment by orndorffgrant** [comment]
@ElijahGeorge @puckybreg added facebook code back in. Requesting final approval. Only test still failing is pharmacy test thats broken.

**Comment by puckybreg** [comment]
👍 Final Approval

**Pull request merged**